index.html

```diff
@@ -142,7 +164,184 @@
                 font-size: smaller;
                 padding: 10px 10px;
             }
+            .main-content{
+                width: auto;
+            }
+            .main-text {
+                font-size: 40px;
+                margin: auto;
+                padding: 10px;
+            }
+            .smaller-text {
+                font-size: smaller;
+                margin: auto;
+                padding: 10px;
+            }
+            }
+        .why-us {
+            background-color: var(--darkblue);
+            padding: 100px 195px;
+        }
+        .why-choose-us {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            flex-wrap: wrap;
+        }
+        .why-choose-us-head {
+            font-size: 40px;
+            font-weight: 700;
+            line-height: 57px;
+            margin: 0;
+            text-transform: uppercase;
+        }
+        .why-choose-us-text {
+            font-weight: 500;
+            line-height: 20px;
+            width: 470px;
+            text-align: center;
+        }
+        .grid-cards {
+            display: grid;
+            grid-template-columns: repeat(3,328px);
```

Comment:
Do not set fixed size to your cards. Let them take as much space as they can. Limit only max width of the container